CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## unreleased
 
   - Update braintree-web to v3.84.0
+  - Update broken create PayPal sandbox references to updated URL
```

Comment:
Since these updates just fix links that once worked, we probably don't need to add a changelog entry for this.